addon.json

```diff
@@ -0,0 +1,74 @@
+{
+    "identifier": "com.seeq.addons.plot_curve",
+    "name": "Plot Curve",
+    "description": "A tool for fitting curves to tabular data and pushing resulting formulas to Seeq",
+    "version": "1.0.0",
+    "license": "Seeq",
+    "icon": "fa fa-edit",
+    "tags": {
+        "maintainer": "Seeq"
+    },
+    "previews": [],
+    "elements": [
+        {
+            "name": "Plot Curve",
+            "identifier": "com.seeq.addons.plot_curve.datalabfunctions",
```

Comment:
Suggest `datalabfunctions` -> `addontool`

Reply:
Fixed in the next push! Though in the documentation it just said to demark it as `.tool` instead of `.addontool`
![image](https://github.com/seeq12/seeq-plot-curve/assets/86431878/725f68b0-ada5-4b8b-b0ae-29b69b4769f0)

Reply:
`.tool` is good.